server/datastore/cached_mysql/cached_mysql_test.go

```diff
@@ -330,3 +330,78 @@ func TestCachedTeamAgentOptions(t *testing.T) {
 	_, err = ds.TeamAgentOptions(context.Background(), testTeam.ID)
 	require.Error(t, err)
 }
+
+func TestCachedTeamFeatures(t *testing.T) {
+	t.Parallel()
+
+	mockedDS := new(mock.Store)
+	ds := New(mockedDS, WithTeamAgentOptionsExpiration(100*time.Millisecond))
```

Comment:
We need a `WithTeamFeaturesExpiration()` option.

Reply:
awesome catch, thanks